main.js

```diff
@@ -0,0 +1,173 @@
+    } else {
+      //apply likes only to selected comment
+      commentsObject.comments[$currentCommentId]['likes']++
+      const $currentLikes = commentsObject.comments[$currentCommentId]['likes']
+      if($(this).parent().children().length === 3) {
+        $(this).parent().append('<i class="col-1 text-end text-primary offset-3 pr-0 bi bi-hand-thumbs-up-fill"></i>')
+        $(this).parent().append('<p class="col-1 text-start text-primary pl-0 mb-0"><small class="likes-text">1<small></p>')  
+      } else {
+        $(this).parent().find('.likes-text').text($currentLikes);
+      }
+    }
+
+  })
+
+
+  //ANCHOR Show Reply Div
+  $($commentContainer).on('click', '.reply-button', function() {
+    const $replyContainer = $(this).parent().siblings('#reply-container')
+    const $newReplyFormContainer = $(this).parent().siblings('#new-reply-form-container')
+    $replyContainer.toggleClass('d-none');
+    $newReplyFormContainer.toggleClass('d-none');
+    const text = $(this).find('small').text();
+    // $(this).find('small').text(text === 'Reply' ? 'Hide' : 'Reply')
+    $(this).toggleClass('bg-danger text-secondary')
+  })
+
+
+  //ANCHOR Add Reply
+  $($commentContainer).on('submit', '#new-reply-form', function(e) {
+    e.preventDefault(); //prevents submission from reloading page
+
+    const $replyName = $(this).find('#reply-name').val()
+    const $replyMessage= $(this).find('#reply-message').val()
+
+    const $currentComment = $(this).closest('.user-comment').attr('id');
+
+    //tracks number of replies in each comment
+    commentsObject.comments[$currentComment]['numberOfReplies']++
+    const $numberOfReplies = commentsObject.comments[$currentComment]['numberOfReplies']
+
+    //populates commentsObject with each reply data
+    commentsObject.comments[$currentComment]['replies'][`reply${$numberOfReplies}`] = {name: $replyName, message: $replyMessage, likes: 0}
+
+    const newReply = $('#new-reply-template').html();
+    const replyTemplate = Handlebars.compile(newReply);
+    const replyHTML = replyTemplate({
+      id: `reply${commentsObject.comments[$currentComment]['numberOfReplies']}`,
+      faker: `${faker.image.cats('100', '100', true)}`, 
+      name: `${$replyName}`,
+      message: `${$replyMessage}`,
+      date: `${getCurrentTime(new Date)}`
+    })
+
+    //updates the reply text to show number of replies within each comment
+    $(this).closest('.user-comment').find('.reply-count').text(` (${$numberOfReplies})`)
+
+    const $replyContainer = $(this).parentsUntil($commentContainer).find('#reply-container')
+    $(replyHTML).hide().appendTo($replyContainer).fadeIn(200);
+
+    $(this)[0].reset();  //resets form after submission
+  })
+
+
+  //ANCHOR Delete Comment on Close Button
+  $($commentContainer).on('click', '.remove-comment', function() {
+    $(this).closest(".comment-reply").remove();
+  })
+})
+
+
+//ANCHOR Function to get current time and display as HH:MM:SS AM/PM
+let getCurrentTime = function(date) {
```

Comment:
This is called a "helper" or "utility" (util) function. It can be a good idea to put these in their own file if you have more than one, just an FYI.